tensorboard/uploader/exporter.py

```diff
@@ -162,8 +180,20 @@ def export(self, read_time=None):
                 else:
                     raise
 
-    def _request_scalar_data(self, experiment_id, read_time):
-        """Yields JSON-serializable blocks of scalar data."""
+    def _request_json_data(self, experiment_id, read_time):
+        """Given experiment id, generates JSON data and destination file name.
+
+        The JSON data describes the run, tag, metadata, in addition to
+          - Actual data in the case of scalars
+          - Pointer to binary files in the case of blob sequences.
```

Comment:
Please document that this method also has the side effect of actually
writing those files (this was not my expectation just from reading the
docs).